.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test_ale:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        vim-version:
+          - '--vim-80-only'
+          - '--vim-81-only'
+          - '--neovim-only'
+          - '--linters-only'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build test docker image
+        uses: docker/build-push-action@v2
+        with:
+          repository: w0rp/ale
+          context: .
+          file: ./Dockerfile
```

Comment:
Seems this is redundant when context is set.

Ref.: https://github.com/docker/build-push-action

Reply:
I removed the docker build step as it takes 5+ minutes that may exhaust the free minutes Github provides monthly.

Reply:
Yeah.  Just noticed that it already exists an image on Docker Hub.